services/management/azure_database.go

```diff
@@ -0,0 +1,376 @@
+			"microsoft.dbforpostgresql/flexibleservers",
+			"microsoft.dbforpostgresql/serversv2":
+			inst.Type = managementpb.DiscoverAzureDatabaseType_DISCOVER_AZURE_DATABASE_TYPE_POSTGRESQL
+		case "microsoft.dbformariadb/servers":
+			inst.Type = managementpb.DiscoverAzureDatabaseType_DISCOVER_AZURE_DATABASE_TYPE_MARIADB
+		default:
+			inst.Type = managementpb.DiscoverAzureDatabaseType_DISCOVER_AZURE_DATABASE_TYPE_INVALID
+		}
+
+		if val, ok := instance.Properties["administratorLogin"].(string); ok {
+			inst.Username = fmt.Sprintf("%s@%s", val, instance.Name)
+		}
+		if val, ok := instance.Properties["fullyQualifiedDomainName"].(string); ok {
+			inst.Address = val
+		}
+		if val, ok := instance.Sku["name"].(string); ok {
+			inst.NodeModel = val
+		}
+
+		resp.AzureDatabaseInstance = append(resp.AzureDatabaseInstance, &inst)
+	}
+
+	return &resp, nil
+}
+
+// AddAzureDatabase add azure database to monitoring.
+//nolint:funlen
+func (s *AzureDatabaseService) AddAzureDatabase(ctx context.Context, req *managementpb.AddAzureDatabaseRequest) (*managementpb.AddAzureDatabaseResponse, error) {
+	l := logger.Get(ctx).WithField("component", "discover/azureDatabase")
+	if e := s.db.InTransaction(func(tx *reform.TX) error {
+		// tweak according to API docs
+		if req.NodeName == "" {
+			req.NodeName = req.InstanceId
+		}
+		if req.ServiceName == "" {
+			req.ServiceName = req.InstanceId
+		}
+
+		// tweak according to API docs
+		tablestatsGroupTableLimit := req.TablestatsGroupTableLimit
+		if tablestatsGroupTableLimit == 0 {
+			tablestatsGroupTableLimit = defaultTablestatsGroupTableLimit
+		}
+		if tablestatsGroupTableLimit < 0 {
+			tablestatsGroupTableLimit = -1
+		}
+
+		// add Remote Azure Database Node
+		node, err := models.CreateNode(tx.Querier, models.RemoteAzureDatabaseNodeType, &models.CreateNodeParams{
+			NodeName:     req.NodeName,
+			NodeModel:    req.NodeModel,
+			AZ:           req.Az,
+			Address:      req.InstanceId,
+			Region:       &req.Region,
+			CustomLabels: req.CustomLabels,
+		})
+		if err != nil {
+			return err
+		}
+		l.Infof("Created Azure Database Node with NodeID: %s", node.NodeID)
+
+		//nolint:exhaustive
+		switch req.Type {
+		case managementpb.DiscoverAzureDatabaseType_DISCOVER_AZURE_DATABASE_TYPE_MYSQL,
+			managementpb.DiscoverAzureDatabaseType_DISCOVER_AZURE_DATABASE_TYPE_MARIADB:
+
+			// add MySQL Service
+			service, err := models.AddNewService(tx.Querier, models.MySQLServiceType, &models.AddDBMSServiceParams{
+				ServiceName:  req.ServiceName,
+				NodeID:       node.NodeID,
+				Environment:  req.Environment,
+				CustomLabels: req.CustomLabels,
+				Address:      &req.Address,
+				Port:         pointer.ToUint16(uint16(req.Port)),
+			})
+			if err != nil {
+				return err
+			}
+			l.Infof("Added Azure Database Service with ServiceID: %s", service.ServiceID)
+
+			azureDatabaseExporter, err := models.CreateAgent(tx.Querier, models.AzureDatabaseExporterType, &models.CreateAgentParams{
```

Comment:
looks like we ignore this field https://github.com/percona/pmm/pull/707/files#diff-544f602d88d0d301b2e1927b963b12eadc7377ca809c9cf6549a015d2340435fR154. Here and in cases below

Reply:
No, qan used for other purposes here.

Reply:
I meant `bool azure_database_exporter`

Reply:
Added.